pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -2244,8 +2244,8 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			},
 				Entry("without kernel args and null container - should approve",
 					"", "", "", "", true, true),
-				Entry("with kernel args and null container - should approve",
-					fakeKernelArgs, "", "", "", true, true),
+				Entry("with kernel args and null container - should reject",
```

Comment:
What about removing the long list of parameters from the function declaration and just having this signature?
```go
func(kernelBoot *v1.KernelBoot, shouldBeValid bool)
``` 
then you can pass a full formed kernelBoot object inside the table entries (even better if you write a helper function to construct those), I think that would be more readable.
Also this way you can get rid of 
```go
vmi := utils.GetVMIKernelBoot()
kb := vmi.Spec.Domain.Firmware.KernelBoot
```
Since  you don't actually need a full VMI spec to unit test this.

Reply:
Thanks for your review! Great feedback.

I've updated the PR with your suggestion :)

Reply:
I would also suggest creating `withoutInitrd`, `withoutKernel`, `withoutImage`